conductor-client/src/main/kotlin/com/openlattice/postgres/external/ExternalDatabaseConnectionManager.kt

```diff
@@ -52,33 +56,61 @@ class ExternalDatabaseConnectionManager(
         return CacheLoader.from { dbName ->
             createDataSource(
                     dbName!!,
-                    assemblerConfiguration.server.clone() as Properties,
+                    assemblerConfiguration.server,
                     assemblerConfiguration.ssl
             )
         }
     }
 
-    fun getOrganizationDatabaseName(organizationId: UUID): String {
-        return organizationDatabases.getValue(organizationId).name
+    fun getDatabaseNameOrDefault(databaseId: UUID, extDbType: ExternalDatabaseType): String {
```

Comment:
I kind of prefer keeping the default name an isolated private function scoped to the class where databases are initialized (which I think is still here?) just since that logic should only ever be executed once at the time of database initialization and never again

Reply:
fixed